### PR TITLE
Add another Node.js course

### PR DESCRIPTION
**Pull request by basuabhirup**
## What does this PR do?
Add resource

### Description
[Node and Express Tutorial](https://www.youtube.com/watch?v=TNV0_7QRDwY) by John Smilga

### Why is this valuable (or not)?
I am personally benefitted from this course. I think this course deserves to get enlisted here.

### How do we know it's really free?
This video tutorial is hosted on YouTube. Just click the link and verify.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

**Comment by basuabhirup** [comment]
Don't know why 2 checks were not successful and what to do in order to pass these 😕

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator. We don't link to authors.

**Comment by eshellman** [comment]
Be sure to check the output of Travis-CI for linter errors. Alphabetical order and spacing are the most common problems.

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -439,6 +439,7 @@
 * [Express JS Crash Course](https://www.youtube.com/watch?v=L72fhGm1tfE) - Brad Traversy, Traversy Media (Youtube)
 * [Node.js API Development for Beginners](https://www.udemy.com/course/node-js-api-tutorial/) (Udemy)
 * [Node.js Basics](https://www.youtube.com/playlist?list=PLqq-6Pq4lTTa-d0iZg41U2RDqECol9C5B) - Java Brains (Youtube)
+* [Node and Express Tutorial](https://www.youtube.com/watch?v=TNV0_7QRDwY) by [John Smilga](https://github.com/john-smilga) - Coding Addict (YouTube)
```
Comment (davorpa) [inline courses/free-courses-en.md:442]: Maybe this solve the linting errors

```
* [Node and Express Tutorial](https://www.youtube.com/watch?v=TNV0_7QRDwY) - John Smilga, Coding Addict (YouTube)
```

Great tutorial @john-smilga

Comment (basuabhirup) [reply courses/free-courses-en.md]: @davorpa I have fixed that error and organised the list properly.

**Review by davorpa — APPROVED** [review]

File: courses/free-courses-en.md (hunk 1 above)
Comment (eshellman) [reply courses/free-courses-en.md]: is there no playlist? is it just one video?

Comment (basuabhirup) [reply courses/free-courses-en.md]: @eshellman No, actually this is an 8-hour long video covering all the aspects. For ease of navigation, they have provided topic-wise timestamps in the video description.

Comment (davorpa) [reply courses/free-courses-en.md]: Maybe the playlist will be this: https://www.youtube.com/playlist?list=PLnHJACx3NwAdT_8forzXYvx0o4A2VnoHX

But this is only have this video

Comment (basuabhirup) [reply courses/free-courses-en.md]: Exactly @davorpa , that playlist contains 10 different videos from 10 different topics 😃

Comment (davorpa) [reply courses/free-courses-en.md]: > Exactly @davorpa , that playlist contains 10 different videos from 10 different topics 😃

I see the video at a glance and for me it's fine. It covers all things. Chapters are attached as timeline blocks as @basuabhirup said.

Eric has the last word 😉

Comment (eshellman) [reply courses/free-courses-en.md]: most YT videos are not, by themselves long enough to qualify as a course. This one certainly is. (8 hours!!!) Probably should add something in the contrib guidelines. 
Thanks, and welcome to free-programming-books!

**Pull request merged**